CHANGES.md

```diff
@@ -6,6 +6,7 @@
 
 - Added `ImageryLayerCollection.pickImageryLayers` which determines the imagery layers that are intersected by a pick ray. [#9651](https://github.com/CesiumGS/cesium/pull/9651)
 - Added a `polylinePositions` getter to `Cesium3DTileFeature` that gets the decoded positions of a polyline vector feature. [#9684](https://github.com/CesiumGS/cesium/pull/9684)
+- Started moving over third party libraries to use npm modules.
 
```

Comment:
Is this the right place to mention the change? I think there should be a note of this somewhere in CHANGES.md.

Reply:
This has zero user facing implications at this time, while we are moving to npm, there are still build time dependencies, not runtime dependencies (i.e. users who use Cesium don't actually have a direct dependencies on these libraries via npm)